doc/lua_api.txt

```diff
@@ -1968,11 +1981,15 @@ Elements
 
 * Show an inventory list if it has been sent to the client. Nothing will
   be shown if the inventory list is of size 0.
+* **Note**: The spacing between inventory slots is not affected by the
+  new coordinate system.
```

Comment:
This is no more true.

Reply:
I'm still tuning this.  I got it totally wrong last time (they were overlapping instead of spaced), so I haven't changed the docs yet.

I decided to change it because even though #7971 can edit the spacing, it will edit it relative to the current values.  I'm currently testing a spacing of half an inventory slot both horizontally and vertically.  This is probably best since it will follow the coordinate system well enough without being too far apart, but I haven't tested it (just started compiling).